src/warnet/ln.py

```diff
@@ -46,9 +49,13 @@ def pubkey(
     print(_pubkey(pod))
 
 
-def _pubkey(pod: str):
-    info = _rpc(pod, "getinfo")
-    return json.loads(info)["identity_pubkey"]
+def _pubkey(pod_name: str):
+    info = _rpc(pod_name, "getinfo")
+    pod = get_pod(pod_name)
+    pubkey_key = "identity_pubkey"
+    if "cln" in pod.metadata.labels["app.kubernetes.io/name"]:
+        pubkey_key = "id"
```

Comment:
this kind of thing is ok for now and probably will be ok for eclair as well -- but its a tad codesmelly. feels like implementation-specific methods should stay in their classes, even as jsut static functions